### DISCUSSION TITLE
Suggestion: show CI checks in the "DETAILS" tab of the commit

### DISCUSSION THREAD

**Discussion by XVilka** [community/community#84938] [Mobile]
### Select Topic Area

Product Feedback

### Body

Currently, there is no way to go from the page of the particular commit to the corresponding CI statuses/checks.
I suggest adding it in the "DETAILS" tab of the screen about the commit.

For example, when I open [this commit](https://github.com/rizinorg/rizin/commit/9094decf4f19dce7d80b05ad288a99717130f29a) in the mobile app, I can see the following screen:

![commit_changes_screen](https://github.com/community/community/assets/203261/e5e08e20-bc77-4cbf-92f8-d2ccea7f7a5d)

You can see the yellow dot of the active CI checks, but pressing to it does nothing as some users might expect. Ok, lets check the "DETAILS" tab:

![commit_details_screen](https://github.com/community/community/assets/203261/9ae48302-11bd-4759-8f0f-70cd5ee09f2b)

As you can see, there is a bunch of free space that can be used to show the checks folded, like it's done for Pull Requests.

**Comment by elisealix22** [comment]
Thank you so much for suggesting this @XVilka ! I opened an issue to track this internally. If you have any other suggestions, please let us know!

**Reply by asportnoy** [reply]
Hi @elisealix22, curious if there's any progress/news on this? Quite annoying to have to open the commit in Safari just to see action details, especially when PR status checks are already built into the app. Thanks!
